packages/cli/package.json

```diff
@@ -81,7 +82,7 @@
     "@types/minimatch": "3.0.3",
     "@types/mri": "1.1.0",
     "@types/ms": "0.7.30",
-    "@types/node": "11.11.0",
+    "@types/node": "16.11.7",
```

Comment:
This version update introduced some type changes throughout the CLI. Might be worth doing this change in a separate PR.